### DISCUSSION TITLE
How to add a map that will unlock if the anomaly happen

### DISCUSSION THREAD

**Discussion by enchanteddaemon** [Lost-MSth/Arcaea-server#112] [Q&A]
![Screenshot_2023-05-20-23-00-52-27_8000ca5dd0dd41d14d6a874a50dd68a3.jpg](https://github.com/Lost-MSth/Arcaea-server/assets/100534753/1a7733c8-152b-4184-8c86-576c66e44c32)

like this

**Comment by Lost-MSth** [comment]
At present, unlocking some world map after completing the specific song is not supported and there is no plan to add this feature in the future. Due to rarity of the anomaly, it is suggested to add some code to handle this event in the score uploading.
